runtime/lua/vim/lsp/_meta.lua

```diff
@@ -2,6 +2,7 @@
 error('Cannot require a meta file')
 
 ---@alias lsp.Handler fun(err: lsp.ResponseError?, result: any, context: lsp.HandlerContext, config?: table): ...any
+---@alias lsp.MultiHandler fun(results: table<integer,{error: lsp.ResponseError?, result: any}>, context: lsp.HandlerContext, config?: table): ...any
```

Comment:
```suggestion
---@alias lsp.MultiHandler fun(results: table<integer,{err: lsp.ResponseError?, result: any}>, context: lsp.HandlerContext, config?: table): ...any
```

Reply:
Yeah, I agree. However, that would be a breaking change.

How do we proceed? Just rip the band aid off and do it?

@mfussenegger

Reply:
An option could be to change the annotation to use `err` and then do something like this for BWC:

```
results[ctx.client_id] = { err = err, error = err, result = result }
```

But no idea how we'd get users to notice that the ought to change it so we could remove it at some point.

Reply:
Could we maybe do something with metatables to warn users when they set `error`?

Reply:
Users won't set `error`; only read it. I think the most we can do is set both, document it, and remove it at some point.

Errors are rare anyway so I think typing messages from LuaLS should give them enough warning.